server/application.py

```diff
@@ -9,14 +9,17 @@
 # Modules that had direct imports (NOT patched): ['urllib3.util, 'urllib3.util.ssl']"
 # which comes from using requests (its deps) lib in webhooks
 
-import os
-from random import randint
+from mergin.config import Configuration as MainConfig
 
-if not os.getenv("NO_MONKEY_PATCH", False):
```

Comment:
Do we want to refactor this NO_MONKEY_PATCH variable in whole system?

Reply:
Yes, I will create PR for it.